azext_edge/edge/params.py

```diff
@@ -694,6 +694,13 @@ def load_iotops_arguments(self, _):
             "--data can be used 1 or more times. Review help examples for full parameter usage",
             arg_group="Additional Info",
         )
+        context.argument(
+            "data_points_file_path",
+            options_list=["--data-file", "--df"],
+            help="File path for the file containing the data points. The following file types are supported: "
```

Comment:
We talked about an aka.ms link pointing to documentation that explains file formats with examples.

Reply:
done